test/smoke/harness/harness.go

```diff
@@ -366,6 +366,21 @@ func (h *Harness) FastForward(d time.Duration) {
 	}
 }
 
+// FastForwardExtended handles time advances of > 30 days refreshing DefaultGraphQLAdminUserID auth which would otherwise be expired.
+func (h *Harness) FastForwardExtended(d time.Duration) {
```

Comment:
Instead of adding a new method, let's just use the existing, and fix the GraphQL code's bug by auto refreshing the session if/when needed (e.g., attempt a single retry upon 401).